build/scenarios.sh

```diff
@@ -158,13 +158,16 @@ jobs=(
   "-n BlazorFormInput $trend $blazorJobs"
 
   # Mono
-  "-n PlaintextPlatform-Mono --webHost KestrelSockets $trend $plaintextPlatformJobs --mono-runtime"
-  "-n Plaintext-Mono --webHost KestrelSockets $trend $plaintextJobs -i 3 --mono-runtime"
-  "-n PlaintextNonPipelined-Mono --webHost KestrelSockets $trend $plaintextJobs --mono-runtime"
-  "-n JsonPlatform-Mono --webHost KestrelSockets $trend $jsonPlatformJobs --mono-runtime"
-  "-n Json-Mono --webHost KestrelSockets $trend $jsonJobs --mono-runtime"
-  "-n Plaintext-Mono -m https --webHost KestrelSockets $trend $plaintextJobs --mono-runtime"
-  "-n DbFortunesRaw-Mono --webHost KestrelSockets $trend $htmlJobs --database PostgreSql --mono-runtime"
+  "-n PlaintextPlatform-Mono --webHost KestrelSockets $trend $plaintextPlatformJobs -i 3"
```

Comment:
We don't use this file anymore. Please take a look at build/trends.scenarios.yml